### ISSUE TITLE
runbot 9.0 red due to letsencrypt?

### ISSUE THREAD

**Issue by sbidoul** [OCA/server-tools#458]
Hi,

It seems the 9.0 branch is red on runbot due to the letsencrypt module?

```
Call of self.pool.get('letsencrypt').cron(cr, uid, *()) failed in Job 2
Traceback (most recent call last):
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3148182-9-0-209efa/openerp/addons/base/ir/ir_cron.py", line 129, in _callback
    getattr(model, method_name)(cr, uid, *args)
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3148182-9-0-209efa/openerp/api.py", line 250, in wrapper
    return old_api(self, *args, **kwargs)
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3148182-9-0-209efa/openerp/api.py", line 354, in old_api
    result = method(recs, *args, **kwargs)
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3148182-9-0-209efa/openerp/addons/letsencrypt/models/letsencrypt.py", line 151, in cron
    account_key, csr, acme_challenge, log=_logger, CA=DEFAULT_CA)
  File "/srv/openerp/instances/openerp-oca-runbot/sandbox/local/lib/python2.7/site-packages/acme_tiny.py", line 104, in get_crt
    raise ValueError("Error requesting challenges: {0} {1}".format(code, result))
ValueError: Error requesting challenges: 400 {
  "type": "urn:acme:error:malformed",
  "detail": "Error creating new authz :: Invalid character in DNS name",
  "status": 400
}
```

@hbrunn

**Comment by hbrunn** [comment]
with https://github.com/OCA/server-tools/pull/459, I get a quite different error. Can you restart your build?

**Comment by hbrunn** [comment]
@sbidoul should be fixed by #464

**Comment by sbidoul** [comment]
thanks @hbrunn
